styles.css

```diff
@@ -0,0 +1,41 @@
+html {
+  min-width: 100%;
+  width: 100%;
+}
+
+.ContactInfoContainer {
+  font-size: 16px;
+}
+
+.Headshot {
+  width: 280px;
+  height: 225px;
+}
+
+.ContactInfoTextMain {
+  font-family: "Oswald", sans-serif;
+}
+
+.NameAndTitles {
+  font-size: 30px;
+}
+
+.CurrentPosition {
+  font-size: 23px;
+}
+
+.EntriesColumn {
+  height: 50vh;
+}
+
+.SocialIcon {
+  width: 32px;
+  height: 32px;
+}
+
+.row {
+  @media (max-width: 481px) {
```

Comment:
this was kind of tricky, idk why bootstrap was trying to add such large negative margins at this viewport, but overriding them did the trick